config/config.go

```diff
@@ -100,14 +102,15 @@ type BuildpackYAML struct {
 
 // Config represents PHP specific configuration options for BuildpackYAML
 type Config struct {
-	Version      string    `yaml:"version"`
-	WebServer    string    `yaml:"webserver"`
-	WebDirectory string    `yaml:"webdirectory"`
-	LibDirectory string    `yaml:"libdirectory"`
-	Script       string    `yaml:"script"`
-	ServerAdmin  string    `yaml:"serveradmin"`
-	Redis        Redis     `yaml:"redis"`
-	Memcached    Memcached `yaml:"memcached"`
+	Version             string    `yaml:"version"`
+	WebServer           string    `yaml:"webserver"`
+	WebDirectory        string    `yaml:"webdirectory"`
+	LibDirectory        string    `yaml:"libdirectory"`
+	Script              string    `yaml:"script"`
+	ServerAdmin         string    `yaml:"serveradmin"`
+	EnableHTTPSRedirect bool      `yaml:"disable_https_redirect"`
```

Comment:
The name of this field seems to contradict its `yaml` tag. Is that by design?

Reply:
No, that's a typo. Sorry. Good catch. I pushed an update and fixed this plus added a test to validate. Let me know if it's easier to resubmit this PR, it looks like there are merge conflicts.